### ISSUE TITLE
unzip2 0.2.5 broken on node 10&11

### ISSUE THREAD

**Issue by rendaw** [RealFaviconGenerator/rfg-api#16]
As far as I can tell - due to unzip2 -> graceful-fs -> natives.  The error is:

```
ReferenceError: internalBinding is not defined
    at fs.js:25:1
    at req_ (/.../source/static/node_modules/natives/index.js:137:5)
    at Object.req [as require] (/.../source/static/node_modules/natives/index.js:54:10)
    at Object.<anonymous> (/.../source/static/node_modules/unzip2/node_modules/graceful-fs/fs.js:1:99)
```

Other packages and unzip 0.2.7 depend on a newer (1 major version) version of graceful-fs.

Edit: unzip 0.2.7 isn't published on npm apparently

This also breaks on node 10 AFAICT, they backported the update that broke native in node 11 to the LTS version.

**Comment by edoardolincetto** [comment]
I've already opened a pull request #15 that references a new version of unzip and could also fix this issue

**Comment by phbernard** [comment]
Fixed in 0.5.0, published a minute ago.
